Add documentation for behaviour of minimum_chrome_version field. (#5289)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/mv3/manifest/minimum_chrome_version/index.md

title: "Manifest - Minimum Chrome Version"
seoTitle: "Chrome Extensions Manifest: minimum_chrome_version"
date: 2013-05-12
updated: 2023-02-14
description: Reference documentation for the minimum_chrome_version property of manifest.json.
---

> An optional manifest key containing a string that defines what versions of Chrome are able to install the extension. The value set for this string must be a substring of an existing Chrome browser version string. Either a full version number can be used if you need to specify a specific update to Chrome, or simply the first number in the string to specify a particular major version.

Reviewer: nit: I think we are now wrapping at 80 chars for these files.  Mind doing that, since we're updating them?

> An optional manifest key containing a string that defines what versions of Chrome are able to install the extension. The value set for this string must be a substring of an existing Chrome browser version string. Either a full version number can be used if you need to specify a specific update to Chrome, or simply the first number in the string to specify a particular major version.

Reviewer: ```suggestion
An optional manifest key containing a string that defines which versions of Chrome are able to install the extension. The value set for this string must be a substring of an existing Chrome browser version string. You can use a full version number to specify a specific update to Chrome, or you can use the first number in the string to specify a particular major version.
```

 ```json
{
  // ...
  "minimum_chrome_version": "107.0.5304.87", // Can also be abbreviated to "107", "107.0", or "107.0.5304"
  // ...
}
```

## Enforcement

### New Installs

> In versions of Chrome older than the minimum version, the Chrome Web Store will show a "Not compatible" message in place of the install button. Users will not be able to install your extension.

Reviewer: ```suggestion
In versions of Chrome older than the minimum version, the Chrome Web Store will show a "Not compatible" message in place of the install button. Users on these versions will not be able to install your extension.
```
